ui/src/components/BaseFormView/BaseFormView.test.tsx

```diff
@@ -66,3 +73,63 @@ it('should pass default values to custom component correctly', async () => {
 
     expect((customModal as HTMLSelectElement)?.value).toEqual('input_three');
 });
+
+it.each([
+    { page: 'configuration', config: getGlobalConfigMockGroupsForConfigPage(), service: 'account' },
+    { page: 'inputs', config: getGlobalConfigMockGroupsFoInputPage(), service: 'demo_input' },
+])('entities grouping for page works properly %s', async ({ config, page, service }) => {
+    setUnifiedConfig(config);
+    const getElementsByGroup = (group: string) => {
+        const firstField = screen.queryByText(`Text 1 Group ${group}`);
+        const secondField = screen.queryByText(`Text 2 Group ${group}`);
+        return { firstField, secondField };
+    };
+    const verfyDisplayedElement = (group: string) => {
+        const { firstField, secondField } = getElementsByGroup(group);
+        expect(firstField).toBeInTheDocument();
+        expect(secondField).toBeInTheDocument();
+    };
+    const verifyNotDisplayedElement = (group: string) => {
+        const { firstField, secondField } = getElementsByGroup(group);
+        expect(firstField).not.toBeInTheDocument();
+        expect(secondField).not.toBeInTheDocument();
+    };
+    render(
+        <BaseFormView
+            page={page as StandardPages}
+            stanzaName={STANZA_NAME}
+            serviceName={service}
+            mode="create"
+            currentServiceState={{}}
+            handleFormSubmit={handleFormSubmit}
+        />
+    );
+    const group1Header = await screen.findByText('Group 1', { exact: true });
+    expect(group1Header).toBeInTheDocument();
+    const group2Header = await screen.findByRole('button', { name: 'Group 2' });
+    expect(group2Header).toBeInTheDocument();
+    const group3Header = await screen.findByRole('button', { name: 'Group 3' });
+    expect(group3Header).toBeInTheDocument();
+    verfyDisplayedElement('1');
+    verfyDisplayedElement('2');
+    verifyNotDisplayedElement('3'); // group 3 is not expanded by default
+    expect(group3Header.getAttribute('aria-expanded')).toEqual('false');
```

Comment:
```suggestion
    expect(group3Header).toHaveAttribute('aria-expanded', 'false');
```
The same for the rest `getAttribute`

Compare:

<img width="1005" alt="image" src="https://github.com/user-attachments/assets/bdfb4923-8987-4a41-8153-fa2b138cd981">

vs

<img width="1365" alt="image" src="https://github.com/user-attachments/assets/27f9ac9a-64f3-4fb6-b091-7b75301d5166">

Reply:
yea, make sense